src/altinn-app-frontend/src/components/base/FileUpload/FileUploadWithTag/FileListComponent.tsx

```diff
@@ -302,12 +230,17 @@ export function FileList(props: FileListProps): JSX.Element {
                                     'form_filler.file_uploader_list_status_done',
                                     props.language,
                                   )}
+                                  style={
+                                    props.mobileView
+                                      ? { marginLeft: '10px' }
+                                      : null
+                                  }
                                 />
                               </div>
-                            )}
+                            ) : null}
                             {!attachment.uploaded && (
```

Comment:
This could be simplified even further - the previous condition is checking the opposite, so instead of rendering `null` here, you can render this condition instead.

```suggestion
                            ) : (
```

Would need to change the rest of the condition as well though on ca line 255

Reply:
Yeah it was not that easy to read the diff for this 😅 I think this component needs some love and simplification. Maybe Ill just do generic code cleanup for my sabbaitcal 🤔🤓

Reply:
Well, turns out this function has a cognitive complexity of 22 - so it's par for the course! :laughing: